04_snake_game/main.go

```diff
@@ -151,7 +211,16 @@ func main() {
 				}
 			}
 		case <-ticker.C:
+			if borderCrash(g) {
+				fmt.Println("GAME OVER")
```

Comment:
optional: Instead of just printing "GAME OVER", could you show a separate game over screen?

Something like

```
+-----------------+
|                 |
|    GAME OVER    |
|                 |
+-----------------+
```